db/seeds.rb

```diff
@@ -1,9 +1,10 @@
-# This file should ensure the existence of records required to run the application in every environment (production,
-# development, test). The code here should be idempotent so that it can be executed at any point in every environment.
-# The data can then be loaded with the bin/rails db:seed command (or created alongside the database with db:setup).
-#
-# Example:
-#
-#   ["Action", "Comedy", "Drama", "Horror"].each do |genre_name|
-#     MovieGenre.find_or_create_by!(name: genre_name)
-#   end
+30.times do |_n|
+  user = User.new(
+    name: Faker::Internet.username.gsub(/\.|_/, ''),
+    email: Faker::Internet.email,
+    password: Faker::Internet.password(min_length: 8)
+  )
+
+  puts user.attributes
+  user.save!
```

Comment:
puts必要なさそうなら、こっちでもいい
```suggestion
  User.create!(
    name: Faker::Internet.username.gsub(/\.|_/, ''),
    email: Faker::Internet.email,
    password: Faker::Internet.password(min_length: 8)
  )
```